cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/IdempotenceCacheTest.java

```diff
@@ -74,7 +74,7 @@ public void testIdempotence() {
     for (MaintenanceEventType eventType : MaintenanceEventType.cachedValues()) {
       parameterConfigOverrides.put(MAINTENANCE_EVENT_TYPE_CONFIG, eventType);
       // Expect mocks.
-      EasyMock.expect(_mockKafkaCruiseControl.config()).andReturn(_config).times(4);
+      EasyMock.expect(_mockKafkaCruiseControl.config()).andReturn(_config).atLeastOnce();
```

Comment:
(The same comment applies to other similar uses) can we avoid dropping `times(...)`?